app/_how-tos/export-an-api-spec-as-a-document.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Export your API design document or request data
+content_type: how_to
+
+products:
+  - insomnia
+
+description: Export a design document, collection, or single request from Insomnia.
+
+tags:
+  - documents
+
+tldr:
+  q: How do I export an API design document, collection, or a request in Insomnia?
+  a: In your workspace, open the Document or Collection menu, click **Preferences → Data**, select the target format and scope, and then export the file.
+
+faqs:
+  - q: Can I export just one request?
+    a: Yes. Click **Export Data → Single Request** from the Document or Collection menu or click  **Preferences → Data**. Then select just that request.
```

Comment:
```suggestion
    a: Select the collection drop-down menu, then **Export**, from here you can select an individual request.
```

Reply:
This is what it looked like on my system: 
<img width="1197" height="658" alt="image" src="https://github.com/user-attachments/assets/eb1935f3-d557-4f7f-8ba6-b6b4e1452088" />

Reply:
I think you can remove this FAQ item in favor of making the entire doc about exporting a design document. 

The requests information could be added in an existing reference doc.